Gems/Atom/Tools/AtomToolsFramework/Code/Source/Inspector/PropertyWidgets/PropertyStringBrowseEditCtrl.cpp

```diff
@@ -47,7 +47,7 @@ namespace AtomToolsFramework
         {
             clearButton->setVisible(true);
             clearButton->setEnabled(true);
-            QObject::connect(clearButton, &QToolButton::clicked, this, &PropertyStringBrowseEditCtrl::Clear);
+            QObject::connect(clearButton, &QToolButton::clicked, this, [this](){ ClearValue(); OnValueChanged(); });
```

Comment:
Nit: For readability
```suggestion
            QObject::connect(clearButton, &QToolButton::clicked, this, [this]() {
                ClearValue();
                OnValueChanged();
            });
```